InterGalacticCurrencyConvertor/src/test/java/DataReaderTest.java

```diff
@@ -0,0 +1,165 @@
+package test.java;
+
+import static org.junit.Assert.*;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import main.java.DataProcessResult;
+import main.java.DataReader;
+
+import org.junit.Test;
+
+/**
+ * Stores test data : the input and the expected output;
+ */
+class Data {
```

Comment:
This is a data container - make it immutable. Same comments for DataProcessResult apply here.

Reply:
updated